app/models/reports/export_raw.rb

```diff
@@ -68,7 +68,7 @@ def default_report_hash
         actual_total: -> (od) { as_currency(od.actual_total) },
         reservation_start_time: -> (od) { od.reservation.reserve_start_at if od.reservation },
         reservation_end_time: -> (od) { od.reservation.reserve_end_at if od.reservation },
-        reservation_minutes: -> (od) { od.reservation.duration_mins if od.reservation },
+        reservation_minutes: -> (od) { od.time_data.try(:duration_mins) },
```

Comment:
I am fully aware there will likely be merge conflicts in this file depending on what PR gets in first.